pyproject.toml

```
@@ -101,7 +101,7 @@ testing = [
     "build~=1.0",
     "jsondiff~=2.0",
     "pytest>=7.2,<9.0",
-    "pytest-cov>=4,<6",
+    "pytest-cov>=4,<7",
```

Is there are reason to keep such a large version range here?
I'd just change these to have the minimum latest version...

@ml-evs?

```suggestion
    "pytest~=8.3",
    "pytest-cov~=6.0",
```

---

Fine by me, we just haven't updated it in a while I guess